test/python/WMCore_t/MicroService_t/Unified_t/Transferor_t.py

```diff
@@ -16,16 +16,18 @@
         RequestStore, RequestManager
```

Comment:
And that's why jenkins report 7 tests deleted. Because you're importing `RequestManager` here, which you renamed to `MSManager`. Please fix it.